.changes/unreleased/Dependencies-20230213-174442.yaml

```
@@ -0,0 +1,6 @@
+kind: Dependencies
+body: Updates cryptography dependency to be <=39.0.1
+time: 2023-02-13T17:44:42.716454Z
+custom:
+  Author: surbias
+  PR: "465"
```

Two things need to be updated here:
1. Body to reflect updated upper bound.
1. We've updated our changie template to ask for issue numbers instead of PR numbers.

So I'll commit these changes:
```suggestion
body: Updates cryptography dependency to be <40.0.0
time: 2023-02-13T17:44:42.716454Z
custom:
  Author: surbias
  Issue: "465"
```